fvm/transactionVerifier.go

```diff
@@ -66,12 +66,16 @@ func (entry *signatureContinuation) verify() errors.CodedError {
 
 	entry.invokedVerify = true
 
+	valid, message := entry.ValidateExtensionDataAndReconstructMessage(entry.payload)
+	if !valid {
+		entry.verifyErr = entry.newError(fmt.Errorf("signature extension data is not valid"))
+	}
+
```

Comment:
a new change in FVM: important as it touches the crypto transaction verification